software/livecodes.yml

```
@@ -0,0 +1,12 @@
+name: "LiveCodes"
+website_url: "https://livecodes.io/docs/features/self-hosting"
+source_code_url: "https://github.com/live-codes/livecodes"
+description: "A feature-rich client-side playground for React, Vue, Svelte, Solid, Typescript, Python, Go, Ruby, PHP and 90+ languages."
```

> 90+ languages

I feel this could be clarified a bit. Is it 90 other programming languages? Real world languages? How to convey that information?

---

I would say its clear as it lists lots of programming languages before stating 90+

---

I think it might be better to do this:
```suggestion
description: "A feature-rich client-side playground for React, Vue, Svelte, Solid, Typescript, Python, Go, Ruby, PHP and 90+ other languages."
```